### PR TITLE
Mark `libarrow* 16.1.0` as broken

### PR DESCRIPTION
**Pull request by vyasr**
The arrow feedstock has run exports set up to allow runtime versions to float between minor versions within the same major family. Arrow did not release any minor versions between 1.0 and 16.0, only major and patch versions, so this export was never leveraged. However, the 16.1 release of revealed that this run export is not valid because arrow libraries are versioned such that different minor versions are not ABI-compatible, so libraries compiled against 16.0 will not be able to use 16.1 at runtime (and vice versa). The recipe is being fixed in https://github.com/conda-forge/arrow-cpp-feedstock/pull/1409

## Guidelines for marking packages as broken:

* We prefer to patch the repo data (see [here](https://github.com/conda-forge/conda-forge-repodata-patches-feedstock))
  instead of marking packages as broken. This alternative workflow makes environments more reproducible.
* Packages with requirements/metadata that are too strict but otherwise work are
  not technically broken and should not be marked as such.
* Packages with missing metadata can be marked as broken on a temporary basis
  but should be patched in the repo data and be marked unbroken later.
* In some cases where the number of users of a package is small or it is used by
  the maintainers only, we can allow packages to be marked broken more liberally.
* We (`conda-forge/core`) try to make a decision on these requests within 24 hours.

What will happen when a package is marked broken?

* Our bots will add the `broken` label to the package. The `main` label will remain on the package and this is normal.
* Our bots will rebuild our repodata patches to remove this package from the repodata.
* In a few hours after the `anaconda.org` CDN picks up the new patches, you will no longer be able to install the package from the `main` channel.

## Checklist:

* [x] I want to mark a package as broken (or not broken):
  * [x] Added a description of the problem with the package in the PR description.
  * [x] Pinged the team for the package for their input.

* [ ] I want to archive a feedstock:
  * [ ] Pinged the team for that feedstock for their input.
  * [ ] Make sure you have opened an issue on the feedstock explaining why it was archived.
  * [ ] Linked that issue in this PR description.
  * [ ] Added links to any other relevant issues/PRs in the PR description.

* [ ] I want to request (or revoke) access to an opt-in CI resource:
  * [ ] Pinged the relevant feedstock team(s)
  * [ ] Added a small description explaining why access is needed

### CODE REVIEW

**Comment by vyasr** [comment]
ping @conda-forge/arrow-cpp

**Comment by h-vetinari** [comment]
Can you please sort the entries? It's hard to see if it's complete

**Comment by vyasr** [comment]
What sort order do you want? Right now there's a top-level sort by the library (e.g. all of libarrow, then all of libarrow-all, etc). Within each library it's ordered by the platform (basically the ordering visible on the anaconda package websites). 

Good catch on parquet, just added it.

**Comment by vyasr** [comment]
I've resorted the packages by alphabetical order, then within each package sorted the different builds alphabetically as well. I've also added a newline between packages so that it's hopefully easier to see the group.

File: requests/arrow161.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
+action: broken
+packages:
+- linux-64/libarrow-16.1.0-he216f8a_0_cuda.conda
```
Comment (jakirkham) [inline requests/arrow161.yml:3]:
```suggestion
# libarrow
- linux-64/libarrow-16.1.0-he216f8a_0_cuda.conda
```

File: requests/arrow161.yml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+- linux-aarch64/libarrow-16.1.0-h23e021c_0_cuda.conda
+- linux-aarch64/libarrow-16.1.0-hda7f696_0_cpu.conda
+- linux-ppc64le/libarrow-16.1.0-h8c6cec4_0_cpu.conda
+- linux-ppc64le/libarrow-16.1.0-had8385d_0_cuda.conda
+- osx-64/libarrow-16.1.0-hb6a69ac_0_cpu.conda
+- osx-arm64/libarrow-16.1.0-h23f55cf_0_cpu.conda
+- win-64/libarrow-16.1.0-h107e38f_0_cpu.conda
+- win-64/libarrow-16.1.0-h1a9115d_0_cuda.conda
+
+- linux-64/libarrow-all-16.1.0-ha770c72_0_cpu.conda
```
Comment (jakirkham) [inline requests/arrow161.yml:14]:
```suggestion
# libarrow-all
- linux-64/libarrow-all-16.1.0-ha770c72_0_cpu.conda
```

File: requests/arrow161.yml [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+- linux-aarch64/libarrow-all-16.1.0-h8af1aa0_0_cpu.conda
+- linux-aarch64/libarrow-all-16.1.0-h8af1aa0_0_cuda.conda
+- linux-ppc64le/libarrow-all-16.1.0-ha3edaa6_0_cpu.conda
+- linux-ppc64le/libarrow-all-16.1.0-ha3edaa6_0_cuda.conda
+- osx-64/libarrow-all-16.1.0-h694c41f_0_cpu.conda
+- osx-arm64/libarrow-all-16.1.0-hce30654_0_cpu.conda
+- win-64/libarrow-all-16.1.0-h57928b3_0_cpu.conda
+- win-64/libarrow-all-16.1.0-h57928b3_0_cuda.conda
+
+- linux-64/libarrow-acero-16.1.0-h530483c_0_cuda.conda
```
Comment (jakirkham) [inline requests/arrow161.yml:25]:
```suggestion
# libarrow-acero
- linux-64/libarrow-acero-16.1.0-h530483c_0_cuda.conda
```

File: requests/arrow161.yml [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+- linux-aarch64/libarrow-acero-16.1.0-h5ad3122_0_cpu.conda
+- linux-aarch64/libarrow-acero-16.1.0-hc1fb278_0_cuda.conda
+- linux-ppc64le/libarrow-acero-16.1.0-h2621725_0_cpu.conda
+- linux-ppc64le/libarrow-acero-16.1.0-hfc8b439_0_cuda.conda
+- osx-64/libarrow-acero-16.1.0-hf036a51_0_cpu.conda
+- osx-arm64/libarrow-acero-16.1.0-h00cdb27_0_cpu.conda
+- win-64/libarrow-acero-16.1.0-he0c23c2_0_cpu.conda
+- win-64/libarrow-acero-16.1.0-he0c23c2_0_cuda.conda
+
+- linux-64/libarrow-dataset-16.1.0-h530483c_0_cuda.conda
```
Comment (jakirkham) [inline requests/arrow161.yml:36]:
```suggestion
# libarrow-dataset
- linux-64/libarrow-dataset-16.1.0-h530483c_0_cuda.conda
```

File: requests/arrow161.yml [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+- linux-aarch64/libarrow-dataset-16.1.0-h5ad3122_0_cpu.conda
+- linux-aarch64/libarrow-dataset-16.1.0-hc1fb278_0_cuda.conda
+- linux-ppc64le/libarrow-dataset-16.1.0-h2621725_0_cpu.conda
+- linux-ppc64le/libarrow-dataset-16.1.0-hfc8b439_0_cuda.conda
+- osx-64/libarrow-dataset-16.1.0-hf036a51_0_cpu.conda
+- osx-arm64/libarrow-dataset-16.1.0-h00cdb27_0_cpu.conda
+- win-64/libarrow-dataset-16.1.0-he0c23c2_0_cpu.conda
+- win-64/libarrow-dataset-16.1.0-he0c23c2_0_cuda.conda
+
+- linux-64/libarrow-flight-16.1.0-hc0a3844_0_cuda.conda
```
Comment (jakirkham) [inline requests/arrow161.yml:47]:
```suggestion
# libarrow-flight
- linux-64/libarrow-flight-16.1.0-hc0a3844_0_cuda.conda
```

File: requests/arrow161.yml [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+- linux-aarch64/libarrow-flight-16.1.0-hae7c30d_0_cpu.conda
+- linux-aarch64/libarrow-flight-16.1.0-he9ec988_0_cuda.conda
+- linux-ppc64le/libarrow-flight-16.1.0-h26e2571_0_cuda.conda
+- linux-ppc64le/libarrow-flight-16.1.0-h3a9f6bb_0_cpu.conda
+- osx-64/libarrow-flight-16.1.0-hd834827_0_cpu.conda
+- osx-arm64/libarrow-flight-16.1.0-h6348776_0_cpu.conda
+- win-64/libarrow-flight-16.1.0-ha7f4a34_0_cpu.conda
+- win-64/libarrow-flight-16.1.0-ha7f4a34_0_cuda.conda
+
+- linux-64/libarrow-flight-sql-16.1.0-h3de2799_0_cuda.conda
```
Comment (jakirkham) [inline requests/arrow161.yml:58]:
```suggestion
# libarrow-flight-sql
- linux-64/libarrow-flight-sql-16.1.0-h3de2799_0_cuda.conda
```

File: requests/arrow161.yml [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+- linux-aarch64/libarrow-flight-sql-16.1.0-h781f5df_0_cpu.conda
+- linux-aarch64/libarrow-flight-sql-16.1.0-h87937b0_0_cuda.conda
+- linux-ppc64le/libarrow-flight-sql-16.1.0-ha51c6cb_0_cuda.conda
+- linux-ppc64le/libarrow-flight-sql-16.1.0-hb85c41f_0_cpu.conda
+- osx-64/libarrow-flight-sql-16.1.0-he1e86a1_0_cpu.conda
+- osx-arm64/libarrow-flight-sql-16.1.0-h6956890_0_cpu.conda
+- win-64/libarrow-flight-sql-16.1.0-hdeef14f_0_cpu.conda
+- win-64/libarrow-flight-sql-16.1.0-hdeef14f_0_cuda.conda
+
+- linux-64/libarrow-gandiva-16.1.0-h35c4161_0_cpu.conda
```
Comment (jakirkham) [inline requests/arrow161.yml:69]:
```suggestion
# libarrow-gandiva
- linux-64/libarrow-gandiva-16.1.0-h35c4161_0_cpu.conda
```

File: requests/arrow161.yml [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+- linux-aarch64/libarrow-gandiva-16.1.0-h1664c5b_0_cuda.conda
+- linux-aarch64/libarrow-gandiva-16.1.0-hde4dc41_0_cpu.conda
+- linux-ppc64le/libarrow-gandiva-16.1.0-hbe4c755_0_cuda.conda
+- linux-ppc64le/libarrow-gandiva-16.1.0-hd5e24c5_0_cpu.conda
+- osx-64/libarrow-gandiva-16.1.0-hfa04d5a_0_cpu.conda
+- osx-arm64/libarrow-gandiva-16.1.0-h31ed65c_0_cpu.conda
+- win-64/libarrow-gandiva-16.1.0-hd4515a1_0_cpu.conda
+- win-64/libarrow-gandiva-16.1.0-hd4515a1_0_cuda.conda
+
+- linux-64/libarrow-substrait-16.1.0-h11d00a1_0_cuda.conda
```
Comment (jakirkham) [inline requests/arrow161.yml:80]:
```suggestion
# libarrow-substrait
- linux-64/libarrow-substrait-16.1.0-h11d00a1_0_cuda.conda
```

File: requests/arrow161.yml [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+- linux-aarch64/libarrow-substrait-16.1.0-h08b7278_0_cpu.conda
+- linux-aarch64/libarrow-substrait-16.1.0-hc89004f_0_cuda.conda
+- linux-ppc64le/libarrow-substrait-16.1.0-h7b16dfb_0_cpu.conda
+- linux-ppc64le/libarrow-substrait-16.1.0-hd482fe8_0_cuda.conda
+- osx-64/libarrow-substrait-16.1.0-h85bc590_0_cpu.conda
+- osx-arm64/libarrow-substrait-16.1.0-hc68f6b8_0_cpu.conda
+- win-64/libarrow-substrait-16.1.0-h1f0e801_0_cpu.conda
+- win-64/libarrow-substrait-16.1.0-h1f0e801_0_cuda.conda
+
+- linux-64/libparquet-16.1.0-h6a7eafb_0_cpu.conda
```
Comment (jakirkham) [inline requests/arrow161.yml:91]:
```suggestion
# libparquet
- linux-64/libparquet-16.1.0-h6a7eafb_0_cpu.conda
```

**Review by jakirkham — COMMENTED** [review]
Would this help with readability?

Agree we are testing the limits of comprehension with these package removals. Might be time to rethink how this is done (though not necessarily in this PR)

**Review by h-vetinari — APPROVED** [review]
Thanks!

**Pull request merged**